sqlglot/expressions.py

```diff
@@ -846,10 +846,14 @@ def alias_column_names(self):
 
     @property
     def selects(self):
+        # For EXPLODE
```

Comment:
What does this comment mean?

Reply:
From what I can tell, the code block below was meant to support EXPLODE expressions

Reply:
Let me see if I can clean this up..